From: Takashi Iwai <tiwai@suse.de>
Subject: kABI workaround for asus-wmi changes
Patch-mainline: Never, kABI workaround
References: bsc#1051510

Move the newly added bool flag to a padding hole with ifdef.

Signed-off-by: Takashi Iwai <tiwai@suse.de>

---
 drivers/platform/x86/asus-wmi.h |    4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

--- a/drivers/platform/x86/asus-wmi.h
+++ b/drivers/platform/x86/asus-wmi.h
@@ -44,8 +44,10 @@ struct quirk_entry {
 	bool store_backlight_power;
 	bool wmi_backlight_power;
 	bool wmi_backlight_native;
-	bool wmi_backlight_set_devstate;
 	bool wmi_force_als_set;
+#ifndef __GENKSYMS__
+	bool wmi_backlight_set_devstate;
+#endif
 	int wapf;
 	/*
 	 * For machines with AMD graphic chips, it will send out WMI event
